client/web/compose/src/views/Public/Pages/Records/View.vue

```diff
@@ -119,7 +139,9 @@ export default {
 
     title () {
       const { name, handle } = this.module
-      return this.$t('page:public.record.view.title', { name: name || handle, interpolation: { escapeValue: false } })
+      const titlePrefix = this.inCreating && this.inEditing ? 'create' : !this.inCreating && this.inEditing ? 'edit' : 'view'
```

Comment:
Not quite, no need to check inEditing, if inCreating is true then you're creating, otherwise if inEditing is true then you're viewing